Source package : ocaml-mlgmpidl
File name       : mlgmpidl-1.3.0.tar.gz
Lookaside URL   : https://src.fedoraproject.org/repo/pkgs/ocaml-mlgmpidl/mlgmpidl-1.3.0.tar.gz/sha512/59482ec5b737700e0c83a3cc3f3820e90dea74084d2fefbae1248fb2cfc988e3efc67b8a68e36fef1bcf6e13db80e21e27bdaaae1edb400ee8d224b62377270f/mlgmpidl-1.3.0.tar.gz
Source URL      : https://github.com/nberth/mlgmpidl/archive/1.3.0/mlgmpidl-1.3.0.tar.gz

source file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/source-mlgmpidl-1.3.0.tar.gz: gzip compressed data, from Unix, original size modulo 2^32 184320
lookaside file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/lookaside-mlgmpidl-1.3.0.tar.gz: gzip compressed data, from Unix, original size modulo 2^32 184320


diff -r source-mlgmpidl-1.3.0.tar.gz-extracted/mlgmpidl-1.3.0/opam/opam lookaside-mlgmpidl-1.3.0.tar.gz-extracted/mlgmpidl-1.3.0/opam/opam
10,13c10,13
<         "CPPFLAGS+=-I%{conf-gmp-paths:incdir}%"  { conf-gmp-paths:incdir != "" }
<         "CPPFLAGS+=-I%{conf-mpfr-paths:incdir}%" { conf-mpfr-paths:incdir != "" }
<         "LDFLAGS+=-L%{conf-gmp-paths:libdir}%"   { conf-gmp-paths:libdir != "" }
<         "LDFLAGS+=-L%{conf-mpfr-paths:libdir}%"  { conf-mpfr-paths:libdir != "" } ]
---
>         "CPPFLAGS+=-I%{conf-gmp:incdir}%"  { conf-gmp:incdir != "" }
>         "CPPFLAGS+=-I%{conf-mpfr:incdir}%" { conf-mpfr:incdir != "" }
>         "LDFLAGS+=-L%{conf-gmp:libdir}%"   { conf-gmp:libdir != "" }
>         "LDFLAGS+=-L%{conf-mpfr:libdir}%"  { conf-mpfr:libdir != "" } ]
23,24c23,24
<   "conf-gmp-paths"
<   "conf-mpfr-paths"
---
>   "conf-gmp" {>= "5"}
>   "conf-mpfr" {>= "4"}
26c26
<   "bigarray-compat"
---
>   "bigarray-compat" { ocaml >= "4.07" }
